src/main/kotlin/no/nav/syfo/pdlpersonhendelse/PdlPersonhendelseService.kt

```diff
@@ -0,0 +1,30 @@
+package no.nav.syfo.pdlpersonhendelse
+
+import no.nav.person.pdl.leesah.Personhendelse
+import no.nav.syfo.application.database.DatabaseInterface
+import no.nav.syfo.domain.PersonIdent
+import no.nav.syfo.metric.COUNT_KAFKA_CONSUMER_PDL_PERSONHENDELSE_UPDATES
+import no.nav.syfo.pdlpersonhendelse.db.updatePersonOversiktStatusNavn
+import no.nav.syfo.personstatus.db.getPersonOversiktStatusList
+import org.slf4j.LoggerFactory
+
+class PdlPersonhendelseService(
+    private val database: DatabaseInterface,
+) {
+    fun handlePersonhendelse(personhendelse: Personhendelse) {
+        if (personhendelse.navn != null) {
+            personhendelse.personidenter.forEach { personIdent ->
+                val personOversiktStatusList = database.getPersonOversiktStatusList(personIdent)
+                if (personOversiktStatusList.isNotEmpty()) {
```

Comment:
Det er vel egentlig ikke nødvendig å sjekke om raden finnes her? Vi kan vel bare sette `navn = null where fnr = personident`, og så er det ikke så farlig om ingenting ble oppdatert? 🤔

Reply:
Forsåvidt sant 🤔 Da sparer vi vel en ekstra spørring mot databasen også for de tilfellene vi faktisk må oppdatere, og heller holde styr på om noe ble oppdatert da 🤔